source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/MessageHub/Models/AvailableOperationReceiptData/ChargePriceCimValidationErrorTextFactoryTests.cs

```diff
@@ -34,22 +35,25 @@ public class ChargePriceCimValidationErrorTextFactoryTests
         [Theory]
         [InlineAutoMoqData]
         public void Create_WithMultipleMergeFields_ReturnsExpectedDescription(
-            ChargePriceCommandBuilder chargePriceCommandBuilder,
+            DocumentDtoBuilder documentDtoBuilder,
             CimValidationErrorTextProvider cimValidationErrorTextProvider,
             ChargePriceOperationDto chargeOperationDto,
             ILoggerFactory loggerFactory)
         {
             // Arrange
-            var chargeCommand = chargePriceCommandBuilder.WithChargeOperation(chargeOperationDto).Build();
+            var document = documentDtoBuilder
+                .WithDocumentType(DocumentType.NotifyPriceList)
```

Comment:
Correct docType would be this one:

```suggestion
                .WithDocumentType(DocumentType.RejectRequestChangeOfPriceList)
```